plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -209,6 +219,7 @@
     - name: Create a started container
       community.general.lxd_container:
         name: mycontainer
+        ignore_volatile_options: True
```

Comment:
```suggestion
        ignore_volatile_options: true
```
Below there are two more.

Reply:
i found two other place and fixed them also

Reply:
done